blueocean-dashboard/src/main/js/creation/github/GithubFlowManager.js

```diff
@@ -96,10 +69,6 @@ export default class GithubFlowManager extends FlowManager {
             this.stateId === STATE.STEP_COMPLETE_SUCCESS;
     }
 
-    savedOrgFolder = null;
-
-    savedPipeline = null;
-
```

Comment:
I believe I saw a reference to a `pipeline` property in another file... might be good to initialize it here.